ConsumerApi.Sdk/Endpoints/Messages/Types/SendMessageRequest.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Backbone.ConsumerApi.Sdk.Endpoints.Messages.Types;
+
+public class SendMessageRequest
+{
+    public List<SendMessageRequestRecipientInformation> Recipients { get; set; } = [];
```

Comment:
Why do you set this default value? Sending a message with no recipients doesn't make any sense. So making the property required should be better. Or is there any reason not to?

Reply:
👍